server/queue/queue-rabbitmq/src/main/java/org/apache/james/queue/rabbitmq/view/cassandra/CassandraMailQueueViewModule.java

```diff
@@ -84,7 +87,9 @@
             " when a mail is dequeued from a mail queue, the record associated with that mail still available in this" +
             " table and will not be deleted immediately regarding to the performance impacts," +
             " but after some scheduled tasks")
-        .options(options -> options)
+        .options(options -> options
+            .compactionOptions(SchemaBuilder.timeWindowCompactionStrategy())
```

Comment:
The time window looks like an important configuration parameter:

```
AND compaction = {'compaction_window_size': '1', 
    				  'compaction_window_unit': 'MINUTES', 
    				  'class': 'org.apache.cassandra.db.compaction.TimeWindowCompacti
```

Reply:
1 minute seems too often

Reply:
How about 1 hour (the default window size used for primary keys?)